package-lock.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "medic",
-  "version": "3.15.0",
+  "version": "3.15.0-FR-bulk-user-upload",
```

Comment:
Do we revert the FR version before merging to master? 
same with `package.json`

cc: @njogz @garethbowen

Reply:
Good catch. It makes sense to do so so making the change